code/modules/materials/Mat_RawMaterials.dm

```diff
@@ -248,16 +248,24 @@
 	default_material = "slag"
 	mat_changename = FALSE
 
+ABSTRACT_TYPE(/obj/item/material_piece/rubber)
+/obj/item/material_piece/rubber
+	setup_material()
+		src.create_reagents(10)
+		reagents.add_reagent("rubber", 10)
```

Comment:
So putting plastic under the rubber material group is probably fine (we're not trying to make a comprehensive taxonomy here after all) but I don't think it's correct to say that plastic _contains_ rubber

Reply:
That's very fair. I did a lot of googling to find out that rubber and plastic often contain the same polymers, but I'll remove the reagent bit.